sections/header.liquid

```diff
@@ -80,6 +85,13 @@
       margin-bottom: {{ section.settings.margin_bottom }}px;
     }
   }
+
+  @media screen and (min-width: 990px) {
```

Comment:
I'll unresolve the conversation so we can get more thoughts :D 

I prefer to keep mobile + tablet with consistent spacing as they look the same (hamburger), instead of aiming to be consistent with other sections that look and behave differently from the header.